### PR TITLE
Bumping version to v6.0.0-alpha3

### PR DESCRIPTION
**Pull request by Tim-Brooks**

### CODE REVIEW

File: core/src/test/java/org/elasticsearch/VersionTests.java [hunk 1]
```diff
@@ -46,30 +46,30 @@
...
-        assertThat(V_6_0_0_alpha2.after(V_5_3_0), is(true));
+        assertThat(V_6_0_0_alpha3.after(V_5_3_0), is(true));
 
-        assertThat(V_5_3_0.onOrAfter(V_6_0_0_alpha2), is(false));
+        assertThat(V_5_3_0.onOrAfter(V_6_0_0_alpha3), is(false));
         assertThat(V_5_3_0.onOrAfter(V_5_3_0), is(true));
-        assertThat(V_6_0_0_alpha2.onOrAfter(V_5_3_0), is(true));
+        assertThat(V_6_0_0_alpha3.onOrAfter(V_5_3_0), is(true));
 
         assertTrue(Version.fromString("5.0.0-alpha2").onOrAfter(Version.fromString("5.0.0-alpha1")));
```
Comment (talevy) [inline core/src/test/java/org/elasticsearch/VersionTests.java]: do we want to add another one of these alpha(N) onOrAfter alpha(N-1) checks? not sure if that is necessary

Comment (tbrooks8) [inline core/src/test/java/org/elasticsearch/VersionTests.java]: I'm not sure. Do you mean like:
```
assertThat(V_6_0_0_alpha3.onOrAfter(V_6_0_0_alpha2), is(true));
```

Comment (talevy) [inline core/src/test/java/org/elasticsearch/VersionTests.java]: yeah. seems reasonable to not need it as well. Although I'm not sure what the rational behind some of the versions chosen in the other tests were.

Comment (jasontedor) [inline core/src/test/java/org/elasticsearch/VersionTests.java]: It is to make sure that the version comparison logic orders alphas, betas, and RCs correctly.

Comment (jasontedor) [inline core/src/test/java/org/elasticsearch/VersionTests.java]: To be clear, I don't think we need anything extra here.

Comment (talevy) [inline core/src/test/java/org/elasticsearch/VersionTests.java]: 👍

**Review by talevy — APPROVED** [review]
LGTM

**Pull request merged**